src/components/Shared/DecisionResult.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import LegislationAdoptedIcon from "../Cards/LegislationCard/LegislationAdoptedIcon";
+import LegislationRejectedIcon from "../Cards/LegislationCard/LegislationRejectedIcon";
+import LegislationInProgressIcon from "../Cards/LegislationCard/LegislationInProgressIcon";
+import { MATTER_STATUS_DECISION } from "../../constants/ProjectConstants";
+import { VOTE_DECISION } from "../../constants/ProjectConstants";
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+interface DecisionResultProps {
+  /**Result of the vote or council decision.  This is an enum value, you can see the enumeration in ProjectConstants */
+  result: string;
+}
+
+const DecisionResult = ({ result }: DecisionResultProps) => {
+  let statusIcon = <div />;
+  switch (result) {
+    case VOTE_DECISION.APPROVE:
+      statusIcon = <LegislationAdoptedIcon />;
+      break;
+    case VOTE_DECISION.REJECT:
+      statusIcon = <LegislationRejectedIcon />;
+      break;
+    case MATTER_STATUS_DECISION.REJECTED:
+      statusIcon = <LegislationRejectedIcon />;
+      break;
+    case MATTER_STATUS_DECISION.ADOPTED:
+      statusIcon = <LegislationAdoptedIcon />;
+      break;
+    case MATTER_STATUS_DECISION.IN_PROGRESS:
+      statusIcon = <LegislationInProgressIcon />;
+      break;
+    default:
+      break;
+  }
```

Comment:
I think this may have been updated but not sure. In the storybook docs I am looking at, the council decision column is using the wrong fields. I.e. using Approve instead of Adopted. May just be a storybook docs thing, not sure.

Reply:
Yeah that was my mistake, fixed it.